automatic/dropbox/tools/Helpers.psm1

```diff
@@ -0,0 +1,25 @@
+
+function Install {
+param(
+    [string] $packageName,
+    [string] $url,
+    [string] $checksum
+)
+    $installerType = 'exe'
+    $validExitCodes = @(0)
+    $silentArgs = '/S'
+    $checksumType = 'sha256';
+
+    Install-ChocolateyPackage $packageName `
+        $installerType `
+        $silentArgs `
+        $url `
+        $url `
```

Comment:
This little guy is possibly wrong here. Url should never be used for both unless it contains native installers for both 32 bit ad 64 bit.